script/Arcueid/Day 1/01_00_ARC01_9B.txt

```diff
@@ -237,7 +238,7 @@ Therefore, please only refer to me as Hisui if you can.
 [sha:df0e8ee545ab2a0742d49e0b1f395cd70accab60]{
 -- Page 536, Offset 2524.
 -- お仕えする方に敬称で呼ばれる使用人の心境を察していただければ幸いです」
-I hope you can understand the feelings of a servant who is called with an honorific by the person she serves."
+I hope you can understand the feelings of a servant who is addressed with an honorific by her master."
```

Comment:
revert

Reply:
I prefer 'addressed', though I agree that 'the person she serves' is better.